### PR TITLE
Added Accord Project details

### LINKED ISSUE #55: Add Accord Project to the 2019 Hacktoberfest Swag List
A good first issue for people celebrating Hacktoberfest!

Per [here](https://github.com/crweiner/hacktoberfest-swag-list/issues/44#issuecomment-535578259), [Accord Project](https://www.accordproject.org/) is participating in Hacktoberfest 2019 by giving away their own swag.

Please add Accord Project to the 2019 list following the formatting in the [CONTRIBUTING.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) file. 

You will need to locate a blog post (or official tweet), the type of swag being offered, and a link to their signup form (or some info on how they are tracking contributors). 

Be sure to [register for Hacktoberest first by clicking here](http://hacktoberfest.digitalocean.com) and submit your pull requests after Oct 1st if you would like it to count for the official Hacktoberfest swag.

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Added necessary details for completing Hacktoberfest with Accord Project.
Resolves #55

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
hey @crweiner, please review this PR. i believe i made the required changes.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories.
+
+  *Participating repositories*
```
Comment (crweiner) [inline README.md]: Please remove this list and instead change it to something like "See the blog post for a list of participating repositories".

File: README.md [hunk 2]
```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories.
```
Comment (crweiner) [inline README.md]: There are [very clear requirements](https://www.accordproject.org/events/hacktoberfest-2019/) for each of the swag opportunities, please spell them out. Follow the rules from the Contributing file and the other companies on the list.

File: README.md [hunk 3]
```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
```
Comment (crweiner) [inline README.md]: You also need to add them in the "Least Involvement to Most Involvement" section below. Please follow the Contributing.md guidelines.

File: README.md (hunk 1 above)
Comment (crweiner) [inline README.md]: Please remove this list and instead change it to something like "See the blog post for a list of participating repositories".

File: README.md [hunk 4]
```diff
@@ -144,10 +175,29 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+- **Swag**: limited edition stickers
+- **How to sign up**: Make pull requests on the participating repositories and respond to us when asked for shipping details.
+- **Notes**: Check out the [official page](https://www.accordproject.org/events/hacktoberfest-2019/) for more information.
+
+#### **Accord Project** (limited edition t-shirt and stickers)
+
+- **Requirements**: either of the three
+    * 1 valid `Difficulty: Challenging` PR
+    * 2 valid `Difficulty: Medium` PR
+    * 5 valid `Difficulty: Starter` PR
```
Comment (crweiner) [inline README.md]: Please remove the ` symbols here as they are used for inline code. You can see how it affects the Jekyll build by [looking at the deploy preview here](https://deploy-preview-86--hacktoberfest-swag-list.netlify.com/)

File: README.md [hunk 5]
```diff
@@ -28,14 +26,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories which needn't necessarily be merged in October.
+
+    * 1 `Difficulty: Challenging` PR or 2 `Difficulty: Medium` PRs or 5 `Difficulty: Starter` PRs: limited edition t-shirt and stickers
```
Comment (crweiner) [inline README.md]: Please remove the ` symbols here as they are used for inline code. You can see how it affects the Jekyll build by [looking at the deploy preview here](https://deploy-preview-86--hacktoberfest-swag-list.netlify.com/)

**Comment by oorjitchowdhary** [comment]
hey @crweiner, i think everything's finally complete

**Comment by crweiner** [comment]
Sorry to keep going back and forth on this. Please make sure that all of the ` symbols are removed and I would probably rebase to clean up this PR (or delete the fork and start again with a fresh PR just to keep everything neat and tidy). Be sure to check the [Netlify deploy preview](https://deploy-preview-86--hacktoberfest-swag-list.netlify.com/) to make sure it all looks right. :+1:

**Comment by oorjitchowdhary** [comment]
Moved to #111

**Pull request closed without merging**